pkg/archive/archive_linux_test.go

```diff
@@ -86,9 +86,8 @@ func checkFileMode(t *testing.T, path string, perm os.FileMode) {
 }
 
 func TestOverlayTarUntar(t *testing.T) {
-	oldmask, err := system.Umask(0)
-	assert.NilError(t, err)
-	defer system.Umask(oldmask)
+	oldmask := unix.Umask(0)
+	defer unix.Umask(oldmask)
```

Comment:
Hm.. these are in the same package; I should probably use `overrideUmask()` here (which also leads to the warning I'm adding).